### ISSUE TITLE
Tag version needs to be validated with composer.json

### ISSUE THREAD

**Issue by weltling** [FriendsOfPHP/pickleweb#40]
Currently there's no validation of the tag with the version in the composer.json. This difference, besides confusing, would cause the versions compiled/packaged be different from what is shown on the pickleweb site or in phpinfo().

This interfere with the pickle issue https://github.com/FriendsOfPHP/pickle/issues/111 , so pickleweb could be using the pickle itself for the triple validation - tag, composer.json and the version in the header file (or any other way for different engines).

**Comment by pierrejoye** [comment]
We should remove the version from composer json and only use the tag.

Rmtools can add it so the binaries archive has this  info.

**Comment by weltling** [comment]
Yeah, but we could still use pickle to check the PHP_EXT_VERSION in the source and compare with the tag, no?

**Comment by pierrejoye** [comment]
Exactly, or better said we need both. Server and client side. Bad version
matching must be rejected

**Comment by pierrejoye** [comment]
Check should still be added. Just in case someone puts it in composer.json. I will do it.
